test/lit.site.cfg.in

```diff
@@ -145,6 +145,10 @@ if "@SWIFT_BACK_DEPLOY_CONCURRENCY@" == "TRUE":
 if "@SWIFT_ENABLE_EXPERIMENTAL_DISTRIBUTED@" == "TRUE":
     config.available_features.add('distributed')
 
+config.swift_freestanding_is_darwin = "@SWIFT_FREESTANDING_IS_DARWIN@" == "TRUE"
+config.swift_enable_dispatch = "@SWIFT_ENABLE_DISPATCH@" in ["TRUE", "ON"]
```

Comment:
I don't think this is necessary. I think that you should just in test/CMakeLists.txt normalize these two flags.

Reply:
Thanks, missed that pattern -- addressed in fe71878